ci/run_cudf_examples.sh

```diff
@@ -9,26 +9,36 @@ trap "EXITCODE=1" ERR
 # Support customizing the examples' install location
 cd "${INSTALL_PREFIX:-${CONDA_PREFIX:-/usr}}/bin/examples/libcudf/" || exit
 
+cd basic || exit
 compute-sanitizer --tool memcheck basic_example
+cd ..
```

Comment:
For future reference, a better way to write the "restore to old directory" logic is with pushd/popd since the pop will automatically track where to cd back to.